CMakeLists.txt

```diff
@@ -10,10 +10,33 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH ON)
 
```

Comment:
@marcpaterno, this PR looks great!  Is it possible to add

```cmake
set(SPDLOG_FMT_EXTERNAL ON)
```

somewhere around here so that the user does not have to supply it at the command line?